Tests/Fixtures/Generated/Colors/swift4/defaults-customName.swift

```diff
@@ -35,18 +35,32 @@ internal struct XCTColors {
 
 // MARK: - Implementation Details
 
-// swiftlint:disable operator_usage_whitespace colon
+// swiftlint:disable operator_usage_whitespace
 internal extension XCTColor {
   convenience init(rgbaValue: UInt32) {
-    let red:   CGFloat = CGFloat((rgbaValue >> UInt32(24)) & UInt32(0xff)) / CGFloat(255.0)
-    let green: CGFloat = CGFloat((rgbaValue >> UInt32(16)) & UInt32(0xff)) / CGFloat(255.0)
-    let blue:  CGFloat = CGFloat((rgbaValue >> UInt32( 8)) & UInt32(0xff)) / CGFloat(255.0)
-    let alpha: CGFloat = CGFloat((rgbaValue              ) & UInt32(0xff)) / CGFloat(255.0)
+    let components = rgbaValue.normalizedRGBAComponents
+    self.init(red: components[0], green: components[1], blue: components[2], alpha: components[3])
+  }
+}
+// swiftlint:enable operator_usage_whitespace
+
+fileprivate extension UInt32 {
```

Comment:
Maybe it's just me and I'm being pedantic, but I don't really like having `extension` on a type as common as `UInt32` to add a feature that is _specific_ to colors and not general to every `UInt32`.
I know that this `extension` here is `fileprivate` so it shouldn't affect the rest of the code, by just by principle, wouldn't it be more logical to have that as a free function instead?

`fileprivate rgbaCompoments(from: UInt32) -> [CGFloat]`